--- origsrc/mp3blaster-3.2.5/nmixer/Makefile.am	2009-01-24 09:25:12.000000000 -0600
+++ src/mp3blaster-3.2.5/nmixer/Makefile.am	2009-03-26 02:08:33.785875000 -0500
@@ -1,7 +1,7 @@
 bin_PROGRAMS	= nmixer
 nmixer_SOURCES	= main.cc
 nmixer_DEPENDENCIES = libnmixer.a
-LDADD           = @NCURSES_LIBS@ @NAS_LIBS@ @LIBNMIXER@
+LDADD           = @LIBNMIXER@ @NCURSES_LIBS@ @NAS_LIBS@
 INCLUDES		= -I/usr/include/ncurses -I$(srcdir) -I$(includedir)
 noinst_LIBRARIES = libnmixer.a
 libnmixer_a_SOURCES = nmixer.cc mixers.cc ossmixer.cc nasmixer.cc nmixer.h \
